integration-enabler-java/src/main/java/com/ca/mfaas/eurekaservice/client/config/Ssl.java

```diff
@@ -17,14 +17,27 @@
 @AllArgsConstructor
 @NoArgsConstructor
 public class Ssl {
-    private boolean verifySslCertificatesOfServices;
+    private Boolean enabled;
+
+    private Boolean verifySslCertificatesOfServices;
```

Comment:
Why reference type? It causes to not check easily cases. 
For example
ApiMediationClientImpl > Boolean.TRUE.equals(sslConfig.getVerifySslCertificatesOfServices())